mix.lock

```diff
@@ -1,3 +1,3 @@
 %{"earmark": {:hex, :earmark, "0.1.17"},
-  "hoedown": {:git, "git://github.com/hoedown/hoedown.git", "2a4cf17c70e6572ed42cdca3ea7ca3e768ed17be", []},
```

Comment:
If you want #git to ignore ONLY local changes to a file:

    $ git update-index --assume-unchanged path/to/some/file

[Via @hashnuke](https://twitter.com/HashNuke/status/612560420021256192).